src/libs/Navigation/AppNavigator/AuthScreens.tsx

```diff
@@ -283,10 +282,6 @@ function AuthScreens() {
 
         App.redirectThirdPartyDesktopSignIn();
 
-        if (lastOpenedPublicRoomID) {
```

Comment:
Should we remove this code entirely here? Are we sure it won't introduce regressions on standalone NewDot?